sseclient.py

```diff
@@ -178,12 +178,10 @@ def parse(cls, raw):
                     msg.data = '%s\n%s' % (msg.data, value)
                 else:
                     msg.data = value
-            elif name == 'event':
-                msg.event = value
-            elif name == 'id':
-                msg.id = value
             elif name == 'retry':
                 msg.retry = int(value)
+            else:
+                setattr(msg,name,value)
```

Comment:
This looks risky to me.  I don't think we want data in the stream messing with arbitrary attributes on the object.  As written, stream data could clobber things like __init__ or the `dump` method on the class.

Reply:
we just need a guard